build.sbt

```diff
@@ -260,16 +260,24 @@ sparkEmrConfigs := List(
 
     //    "spark.kryoserializer.buffer.max" -> "2047m",
 
-    //     Best practice 4: Always set up a garbage collector when handling large volume of data through Spark.
+    // Best practice 4: Always set up a garbage collector when handling large volume of data through Spark.
     // Use these GC strategy to avoid java.lang.OutOfMemoryError: GC overhead limit exceeded
      "spark.executor.defaultJavaOptions" -> "-XX:+UseG1GC -XX:+UnlockDiagnosticVMOptions -XX:+G1SummarizeConcMark -XX:InitiatingHeapOccupancyPercent=35 -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:OnOutOfMemoryError='kill -9 %p'",
      "spark.driver.defaultJavaOptions" -> "-XX:+UseG1GC -XX:+UnlockDiagnosticVMOptions -XX:+G1SummarizeConcMark -XX:InitiatingHeapOccupancyPercent=35 -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:OnOutOfMemoryError='kill -9 %p'",
 
-    // set this environment variable for GDAL to use request payer method for S3 files
-    "spark.yarn.appMasterEnv.AWS_REQUEST_PAYER" -> "requester",
+    // set this environment variable for GDAL to use request payer method for S3 files.
+    // Added some of these to resolve 503 errors using 200 worker instances.
     "spark.executorEnv.AWS_REQUEST_PAYER" -> "requester",
+    "spark.yarn.appMasterEnv.AWS_REQUEST_PAYER" -> "requester",
+    "spark.executorEnv.GDAL_HTTP_MAX_RETRY" -> "3",
+    "spark.yarn.appMasterEnv.GDAL_HTTP_MAX_RETRY" -> "3",
+    "spark.executorEnv.GDAL_DISABLE_READDIR_ON_OPEN" -> "EMPTY_DIR",
+    "spark.yarn.appMasterEnv.GDAL_DISABLE_READDIR_ON_OPEN" -> "EMPTY_DIR",
+    "spark.executorEnv.GDAL_HTTP_RETRY_DELAY" -> "10",
+    "spark.yarn.appMasterEnv.GDAL_HTTP_RETRY_DELAY" -> "10"
```

Comment:
For GFWPro analyses, GDAL_HTTP_MAX_RETRY is set to 10 - you may want to change to that for consistency.  @dagibbs22

Reply:
Thanks. Changed.